### ISSUE TITLE
The Colab Tutorial file Error

### ISSUE THREAD

**Issue by Solar0214** [open-mmlab/mmaction2#1253]
Thanks for your error report and we appreciate it a lot.
If you feel we have help you, give us a STAR! :satisfied:

**Checklist**

1. I have searched related issues but cannot get the expected help.
2. The bug has not been fixed in the latest version.

**Describe the bug**

**Reproduction**

1. What command or script did you run?

``` On the MMaction_tutorial colab file:

# Use the recognizer to do inference
video = '/content/mmaction2/demo/demo.mp4'
label = '/content/mmaction2/tools/data/kinetics/label_map_k400.txt'
results = inference_recognizer(model, video, label)

AttributeError: 'Recognizer2D' object has no attribute '/content/mmaction2/tools/data/kinetics/label_map_k400'

During handling of the above exception, another exception occurred:

AttributeError: Module /content/mmaction2/tools/data/kinetics/label_map_k400.txt not found

A placeholder for the command.
```

2. Did you make any modifications on the code or config? Did you understand what you have modified?
``` I have modified the path of the files but those are same as default on the colab tutorial file.

3. What dataset did you use?
```` I just followed the instruction one by one.
**Environment**

1. Please run `PYTHONPATH=${PWD}:$PYTHONPATH python mmaction/utils/collect_env.py` to collect necessary environment information and paste it here.
2. You may add addition that may be helpful for locating the problem, such as
    - How you installed PyTorch [e.g., pip, conda, source]
    - Other environment variables that may be related (such as `$PATH`, `$LD_LIBRARY_PATH`, `$PYTHONPATH`, etc.)

**Error traceback**

If applicable, paste the error traceback here.

``` AttributeError                            Traceback (most recent call last)
/content/mmaction2/mmaction/core/hooks/output.py in register(self, module)
     43                 try:
---> 44                     layer = rgetattr(module, name)
     45                     h = layer.register_forward_hook(hook_wrapper(name))

6 frames
/content/mmaction2/mmaction/core/hooks/output.py in rgetattr(obj, attr, *args)
     67 
---> 68     return functools.reduce(_getattr, [obj] + attr.split('.'))

/content/mmaction2/mmaction/core/hooks/output.py in _getattr(obj, attr)
     65     def _getattr(obj, attr):
---> 66         return getattr(obj, attr, *args)
     67 

/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py in __getattr__(self, name)
    947         raise AttributeError("'{}' object has no attribute '{}'".format(
--> 948             type(self).__name__, name))
    949 

AttributeError: 'Recognizer2D' object has no attribute '/content/mmaction2/tools/data/kinetics/label_map_k400'

During handling of the above exception, another exception occurred:

AttributeError                            Traceback (most recent call last)
/content/mmaction2/mmaction/core/hooks/output.py in register(self, module)
     45                     h = layer.register_forward_hook(hook_wrapper(name))
     46                 except AttributeError:
---> 47                     raise AttributeError(f'Module {name} not found')
     48                 self.handles.append(h)
     49 

AttributeError: Module /content/mmaction2/tools/data/kinetics/label_map_k400.txt not found
A placeholder for traceback.
```

**Bug fix**

If you have already identified the reason, you can provide the information here. If you are willing to create a PR to fix it, please also leave a comment here and that would be much appreciated!

**Comment by kennymckormick** [comment]
I cannot reproduce this bug in the latest colab tutorial, did you use the latest colab tutorial?

**Comment by Solar0214** [comment]
Yes I just clicked the tutorial link on the main github webpage. Is there anywhere else I can get the lastest??

**Comment by Solar0214** [comment]
I just find out what mmaction version is 0.20.0. I guess I got the lastest colab turotial. But still the bug ouccurs. Help :'(

**Comment by kennymckormick** [comment]
Oh, I got it. The colab tutorial is not updated after we refactor the inference API in #1191, I will take time to fix it this weekend. If you know how to update the colab tutorial after reading the #1191 PR, you are also highly encouraged to raise a PR to fix it.

**Comment by Solar0214** [comment]
thank you so much for your help =) I really appreciated!

**Comment by kennymckormick** [comment]
The problem is fixed in #1256, you can have a try.
